python/cudf/cudf/_lib/cpp/binning.pxd

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2021, NVIDIA CORPORATION.
+
+from libcpp.memory cimport unique_ptr
+
+from cudf._lib.cpp.column.column cimport column
+from cudf._lib.cpp.column.column_view cimport column_view
+
+cdef extern from "cudf/binning/bin.hpp" namespace "cudf::bin" nogil:
```

Comment:
Based on the updates to your C++ code qe should also change this to 

`cdef extern from "cudf/labeling/label_bins.hpp" namespace "cudf" nogil:`